modules/services/fluidsynth.nix

```
@@ -21,6 +21,15 @@ in {
         '';
       };
 
+      soundService = mkOption {
+        type = types.str;
```

Perhaps it would be good to have an enum here instead? Something like
```suggestion
        type = types.enum [ "pulseaudio" "pipewire-pulse" ];
```

---

Ok, I went ahead and added that (also including `jack`).